readme.md

```diff
@@ -509,7 +558,49 @@ Any `Defaults` key changes made within the closure will not propagate to `Defaul
 
 Get/set a `Defaults` item and also have the view be updated when the value changes.
 
-This is only implemented for `Defaults.Key`. PR welcome for `Defaults.NSSecureCoding` if you need it.
+### `Defaults.migration(keys..., to: Version)`
+
+```swift
+Defaults.migration<T: Defaults.Serializable & Codable>(keys..., to: Version)
+Defaults.migration<T: Defaults.NativeType>(keys..., to: Version)
+```
+
+Type: `func`
+
+Migrate the given keys to the specific version.
+
+You can specify up to 10 keys. If you need to specify more, call this method multiple times.
+
+#### `Defaults.NativeType`
+
+```swift
+protocol DefaultsNativeType: Defaults.Serializable {
+	associatedtype CodableForm: Defaults.CodableType
+}
+```
+
+Type: `protocol`
+
+Represents the type after migration.
```

Comment:
It doesn't just represent that. It also means the type is natively supported by userdefaults. I think the description should also describe its purpose.